wagtail_wordpress_import/block_builder_defaults.py

```diff
@@ -238,16 +238,16 @@ def get_or_save_image(src):
 def fetch_url(src, r=None, status=False, content_type=None):
     """general purpose url fetcher with ability to pass in own config"""
     try:
-        r = requests.get(src, **conf_get_requests_settings())
-        status = r.status_code == 200
+        response = requests.get(src, **conf_get_requests_settings())
+        status = True if response.status_code == 200 else False
         content_type = (
-            r.headers["content-type"].lower() if r.headers.get("content-type") else ""
+            response.headers["content-type"].lower()
+            if response.headers.get("content-type")
+            else ""
         )
-    except requests.ConnectTimeout:
-        print(f"CONNECTION TIMEOUT: {src}")
-    except requests.ConnectionError:
-        print(f"CONNECTION ERROR: {src}")
-    return r, status, content_type
+        return response, status, content_type
+    except Exception as e:
+        raise requests.ConnectionError(e)
```

Comment:
There seems to be a few possible exception types from requests and to handle all the possible values might be overkill but we do need to avoid the import process from stopping for any exception.

We might change this when we do the ticket for improving the logging (there's no logging for this here other than in the console).

Reply:
Okay, so why catch and reraise? Is it because you want to raise a single error type that you can watch for later?

Reply:
@nickmoreton Can you please address this comment?

Reply:
> Okay, so why catch and reraise? Is it because you want to raise a single error type that you can watch for later?

Yes I was thinking in the later logging improvements we'd log a specific error